cfy_cluster_setup/main.py

```diff
@@ -253,10 +268,10 @@ def _create_cluster_install_directory():
     logger.info('Creating `{0}` directory'.format(DIR_NAME))
     if exists(CLUSTER_INSTALL_DIR):
         new_dirname = (time.strftime('%Y%m%d-%H%M%S_') + DIR_NAME)
-        os.rename(CLUSTER_INSTALL_DIR, join(TOP_DIR, new_dirname))
+        sudo(['mv', CLUSTER_INSTALL_DIR, join(TOP_DIR, new_dirname)])
         for dir_name in os.listdir(TOP_DIR):  # Delete old dir
             if (DIR_NAME in dir_name) and (dir_name < new_dirname):
-                shutil.rmtree(join(TOP_DIR, dir_name))
+                sudo(['rm', '-rf', join(TOP_DIR, dir_name)])
```

Comment:
Is there a way we can avoid using sudo for these steps to reduce the impact of any programming errors that we might introduce at any point in the future?

Reply:
(it should be guarded against by the above checks for now, but I'm having uncomfortable visions of a 'sudo rm -rf /tmp')

Reply:
I absolutely agree with this comment. The thing is that I modified the code to use `sudo` after the previous code raised an authorization error when trying to modify the `/tmp` folder. Is there any way I can modify it to not use `sudo`?

Reply:
What files in the /tmp folder are not owned by the user this script is running as?